app/src/main/res/values/strings.xml

```diff
@@ -445,8 +445,8 @@
     <string name="play_queue_stream_detail">Details</string>
     <string name="play_queue_audio_settings">Audio Settings</string>
     <string name="hold_to_append">Hold to enqueue</string>
-    <string name="enqueue_on_background">Enqueue in the background</string>
-    <string name="enqueue_on_popup">Enqueue in a popup</string>
+    <string name="enqueue_stream">Enqueue</string>
+    <string name="player_append">Enqueued</string>
```

Comment:
Just use `enqueued` as the id, or is it already used?

Reply:
No, it's not used, I've just oriented on the other resources ;)